lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -356,23 +356,23 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
 
         % if is_course_blocked:
           <p id="block-course-msg" class="course-block">
-            ${_("You can no longer access this course because payment has not yet been received. "
+            ${Text(_("You can no longer access this course because payment has not yet been received. "
                 "You can {contact_link_start}contact the account holder{contact_link_end} "
                 "to request payment, or you can "
                 "{unenroll_link_start}unenroll{unenroll_link_end} "
-                "from this course").format(
-              contact_link_start='<a href="#">',
-              contact_link_end='</a>',
-              unenroll_link_start=(
+                "from this course")).format(
+              contact_link_start=HTML('<a href="#">'),
+              contact_link_end=HTML('</a>'),
+              unenroll_link_start=HTML(
                 '<a id="unregister_block_course" rel="leanModal" '
                 'data-course-id="{course_id}" data-course-number="{course_number}" data-course-name="{course_name}" '
                 'href="#unenroll-modal">'.format(
-                  course_id=escape(course_overview.id),
-                  course_number=escape(course_overview.number),
-                  course_name=escape(course_overview.display_name_with_default_escaped),
+                  course_id=course_overview.id,
```

Comment:
@robrap Lines 370-372 need to be escaped, correct?  But they are wrapped within an unescaping `HTML` call.  Any recommendations on how to do so?